src/Engine/CombatParameters.java

```diff
@@ -43,9 +43,9 @@ public void swap() {
 	
 	private void calculateParameters() {
 		baseDamage			= attacker.getDamage(defender);
-		attackFactor		= attacker.model.COStr;
+		attackFactor		= attacker.model.getDamageRatio();
```

Comment:
I think I like the difference of nomenclature: the variable being named "COstr" or "COdef" makes it clear that it's a variable determined by their commander, and is thus effectively external to the unit.
Naming the function (get/modify)(Damage/Defense)Ratio makes it clear the exact function of the variable as it interacts with other units.